torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -2841,13 +2841,22 @@ at::Tensor XLANativeFunctions::threshold_backward(const at::Tensor& grad_output,
       threshold.to<double>()));
 }
 
-std::tuple<at::Tensor, at::Tensor> XLANativeFunctions::topk(
-    const at::Tensor& self, int64_t k, int64_t dim, bool largest, bool sorted) {
-  TORCH_LAZY_FN_COUNTER("xla::");
-  auto results = tensor_methods::topk(bridge::GetXlaTensor(self), k, dim,
-                                      largest, sorted, /*stable=*/false);
-  return std::make_tuple(bridge::AtenFromXlaTensor(std::get<0>(results)),
-                         bridge::AtenFromXlaTensor(std::get<1>(results)));
+std::tuple<at::Tensor, at::Tensor> XLANativeFunctions::topk_symint(
```

Comment:
hmm, will static topk got dispatched to this function too? Seems like it since our topk test doesn't break. I guess upstream just check if downstream implemented `topk` or `topk_symint` ?

Reply:
nvm, now I remember we have a `symint` field in the yaml file.

Reply:
That's right. I basically followed the expand_symint example.